src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsParser.cs

```diff
@@ -227,12 +218,11 @@ internal void TdsLogin(
                            length,
                            feOffset,
                            clientInterfaceName,
-                           outSSPIBuff,
-                           outSSPILength);
+                           sspiWriter is { } ? sspiWriter.WrittenSpan : ReadOnlySpan<byte>.Empty);
 
-            if (rentedSSPIBuff != null)
+            if (sspiWriter is { })
```

Comment:
I believe the `is not null` has more readability over this pattern in the cases like this.